sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_polling.py

```diff
@@ -6,6 +6,16 @@
 import time
 
 from azure.core.polling import PollingMethod
+from azure.keyvault.certificates import KeyVaultCertificate, CertificateOperation
```

Comment:
We can move these inside the `if TYPE_CHECKING:` block since they're only referenced in typehints